doc/interesting-queries.md

```diff
@@ -69,15 +69,15 @@ retire. Therefore to get the latest pool registration for every pool that is sti
 valid:
 ```
 cexplorer=# select * from pool_update
-              where registered_tx_id in (select max(registered_tx_id) from pool_update group by reward_addr)
-              and not exists (select * from pool_retire where pool_retire.hash_id = pool_update.id);
+              where registered_tx_id in (select max(registered_tx_id) from pool_update group by hash_id)
+              and not exists (select * from pool_retire where pool_retire.hash_id = pool_update.hash_id and pool_retire.retiring_epoch <= (select max(no) from epoch));
 
```

Comment:
This query will only work for the "extended" version of `db-sync`, because the non-extended version does not populated the `epoch` table.

Reply:
It can probably be safely be changed to `select max(epoch_no) from block` .

That query run on its own runs pretty much instantaneously.